.github/workflows/ci.yml

```diff
@@ -59,9 +59,9 @@ jobs:
           command: build
           args: --target ${{ matrix.target }} --release
       - name: End to end tests
-        run: |
+        run: >-
```

Comment:
https://stackoverflow.com/questions/15540635/what-is-the-use-of-the-pipe-symbol-in-yaml
https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-over-multiple-lines